### PR TITLE
magento/merchdocs#1160 Add a new topic about adding products to the categories

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds a new child page for the Products in Category page and describes how to add a new product to a category.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/categories-category-products.html

whatsnew
Added the [Adding Category Products](https://docs.magento.com/user-guide/catalog/category-products-add.html) topic to describe adding products to a category from the _Products in Category_ section.

### CODE REVIEW

File: src/catalog/category-products-add.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Adding Category Products
+---
+
+For any category, in Magento 2, the Admin can add products that will be displayed in it. The **Products in Category** section lists all the products that are actually assigned to the category. The <span class="btn">Add Product</span> is displayed if the **Match products by rule** set to **No**.
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:5]:
```suggestion
From the [Products in Category]({% link catalog/categories-category-products.md %}) section , a store administrator can add products to the category. This section lists all the products that are actually assigned to the category and displays <span class="btn">Add Product</span> when **Match products by rule** is set to `No`.
```

File: src/catalog/category-products-add.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+![]({% link images/images-ee/category-product.png %}){: .zoom}
+_Products in Category_
+
+To add products use either the **Search All Products** or **Add Product by SKU** method.
+
+## Method 1: Search All Products
+
+1. Click <span class="btn">Add Products</span> and do the following:
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:14]:
```suggestion
1. Click <span class="btn">Add Products</span>.
```

File: src/catalog/category-products-add.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Use Search by keyword or filters to find and select the products you need.
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:16]:
```suggestion
1. Use _Search by keyword_ or filters to find the products you want to add.
```

File: src/catalog/category-products-add.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    ![]({% link images/images-ee/search-all-product.png %}){: .zoom}
+    _Search All Products Tab_
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:19]:
```suggestion
    _Search All Products tab_
```

File: src/catalog/category-products-add.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+1. To add a product, set the **Assign option** to **Yes** and click <span class="btn">Save and Close</span>.
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:21]:
```suggestion
1. In the _Assign_ column, toggle the option to **Yes** for each product that you want to add.

   If you want to include all of the displayed products, you can click the menu arrow in the column header and choose **Select All**.

1. To apply your changes, click <span class="btn">Save and Close</span>.
```

File: src/catalog/category-products-add.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+### Actions
+
+|Action|Description|
+|--- |--- |
+|Select All|Marks the checkbox of all records in the list.|
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:27]:
```suggestion
|Select All|Selects the checkbox of all records in the list.|
```

File: src/catalog/category-products-add.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+|Unselect All|Clears the checkbox of all records in the list.|
+|Select All on This Page|Marks the checkbox of records on the current page.|
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:29]:
```suggestion
|Select All on This Page|Selects the checkbox of records on the current page.|
```

File: src/catalog/category-products-add.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+|Deselect All on This Page|Clears the checkbox of  records on the current page.|
+
+## Method 2: Add Products by SKU
+
+1. Click <span class="btn">Add Products</span> select **Add Products by SKU** tab.
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:34]:
```suggestion
1. Click <span class="btn">Add Products</span>

1. Select the **Add Products by SKU** tab.
```

File: src/catalog/category-products-add.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+1. Enter SKUs below, one per line, and click <span class="btn">Assign</span>. To discard the changes, click <span class="btn">Remove</span>.
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:36]:
```suggestion
1. Enter SKUs below, one per line, and click <span class="btn">Assign</span>.

   To discard the changes, click <span class="btn">Remove</span>.
```

File: src/catalog/category-products-add.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+    ![]({% link images/images-ee/add-product-by-sku.png %}){: .zoom}
+    _Add Products by SKU tab_
+
+1. For approve changes click <span class="btn">Save and Close</span>.
```
Comment (leslietilling) [inline src/catalog/category-products-add.md:41]:
```suggestion
1. To apply your changes, click <span class="btn">Save and Close</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Some editorial changes and structuring changes for procedures.

File: src/catalog/category-products-add.md (hunk 10 above)
Comment (ZhovnirMaria) [reply src/catalog/category-products-add.md]: @leslietilling 
Thank you, done.

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1161`.

**Pull request merged**